front-end/src/pages/ProposalDetail/ProposalDetail.js

```diff
@@ -0,0 +1,512 @@
+})(Button);
+
+const UploadSmallTypographyCreate = withStyles({
+    root: {
+        fontSize: '15px',
+        color: '#1261A3',
+    }
+})(Typography);
+
+const StatusTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(89, 156, 109, 1)',
+        lineHeight: '14.06px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const MainTitleTyography = withStyles({
+    root: {
+        fontSize: '20px',
+        color: 'rgba(18, 97, 163, 1)',
+        lineHeight: '26.6px',
+        fontWeight: '600'
+    }
+})(Typography);
+
+const GovernmentTitleTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(0, 0, 0, 0.5393)',
+        lineHeight: '16x',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const GovernmentContentMiddleTyography = withStyles({
+    root: {
+        fontSize: '14px',
+        color: 'rgba(0, 0, 0, 1)',
+        lineHeight: '16.41px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const GovernmentContentSmallTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(0, 0, 0, 1)',
+        lineHeight: '16.41px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const CollapseTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(18, 97, 163, 1)',
+        lineHeight: '16px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const CatergoryItemTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(1, 1, 1, 1)',
+        lineHeight: '14.06px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const UploadLock = withStyles({
+    root: {
+        color: '#1261A3'
+    }
+})(Lock);
+
+const CategoryPeopleIcon = withStyles({
+    root: {
+        color: '#000000',
+        fontSize: 15,
+    }
+})(NaturePeople);
+
+export default function ProposalDetail() {
+    const { proposal_id } = useParams();
+
+    const classes = useStyles();
+
+    const [valueBudget, setValueBudget] = useState(20000);
+
+    const [state, setState] = useState({
+        type: 'new',
+    });
+    
+    const [description, setDescription] = useState({
+        content: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing rl Lorem ipsum Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit ame"
+    });
+
+    const [title, setTitle] = useState("")
+
+    const [showHistory, setShowHistory] = useState(true)
+
+    const { errors, handleSubmit } = useForm({
+        criteriaMode: "all"
+    });
+
+    const onSubmit = data => {
+
```

Comment:
Can you add a console.log function so that in future we will come to know where and when this executes ?